cypress/integration/channel/message_spec.js

```diff
@@ -70,4 +70,26 @@ describe('Message', () => {
             cy.focused().should('contain', 'AA');
         });
     });
+
+    it('M14320 @here., @all. and @channel. (ending in a period) still highlight', () => {
+        // 1. Login and go to /
+        cy.login('user-1');
+        cy.visit('/');
+
+        // 2. Post message
+        cy.postMessage('@here. @all. @channel.');
```

Comment:
I've got a failing test since after step 2, the post requires confirmation before it gets actually posted (see below).  Please add action to confirm it.

<img width="450" alt="screen shot 2019-03-08 at 6 09 17 pm" src="https://user-images.githubusercontent.com/5334504/54022650-f2da3a80-41cd-11e9-98ba-91236529c56c.png">